components/Ui/UiCompactFeature.vue

```diff
@@ -1,11 +1,19 @@
 <template>
   <article class="ui-compact-feature">
-    <nuxt-img class="ui-compact-feature__icon" :src="`/images/icons/${icon}`" />
-    <h3 class="h4 ui-compact-feature__title">
-      {{ title }}
-    </h3>
-    <div class="ui-compact-feature__description">
-      {{ description }}
+    <div>
```

Comment:
Since I'm using CSS flex, I used this div to group the content of the card, separately from the detail of the card.

the content div includes the icon, title, and description
the detail div is separate
and w/ css flex, I justify w/ space-between, so that the project count aligns at the bottom, across all 3 cards

otherwise, the other approach to align the project count across 3 cards, was to position absolute...

Reply:
is it not enough grouping it with the `<article>`?

Reply:
@y4izus it's not... the `<article>` _is_ the card itself.

- Within the card, we want to have the project count aligned across all 3 cards (unless I'm wrong about this)
  - the way we're doing that is CSS flex
  - to push the project count (`.__detail` div) to the bottom and have it aligned, we justify space-between
  - the `space-between` separates the 'content' (icon, title, and description), from the 'detail'
  
This is what happens if I remove the `<div>`... 

https://github.com/Qiskit/qiskit.org/assets/6276074/5d23c4a9-4cc7-4aa5-a648-0efb99252875

If there's a better way of achieving this and I'm just not seeing it, I'm open to suggestions

Reply:
correct me if I'm wrong: when you remove `<div>`, the only thing that is not aligned is the icon, why don't you use `align-self: start;` on it?

Reply:
ok, don't spend more time with this, we can merge ;)